rust_src/remacs-macros/lib.rs

```diff
@@ -24,6 +24,12 @@ pub fn lisp_fn(attr_ts: TokenStream, fn_ts: TokenStream) -> TokenStream {
     let mut rargs = quote::Tokens::new();
     let mut body = quote::Tokens::new();
     let max_args = function.args.len() as i16;
+    let intspec = if let Some(intspec) = lisp_fn_args.intspec {
+        quote!{ (#intspec).as_ptr() as *const ::libc::c_char }
```

Comment:
Don't we need a null byte here?

Reply:
Good catch. I incorrectly thought that was being handled by the lisp_attr parsing, but it is not.